### PR TITLE
[CRM] Replace fdbclear with swssconfig DEL

### PR DESCRIPTION
**Pull request by skshih**
### Description of PR

During the fdb test, a testing FDB record is added with swssconfig but removed with fdbclear.

However with fdbclear the record is not removed from appDB after testing.

The remaining record in appDB may result in the next test case "warm-reboot" to fail 

becauses orchagent restore failed and terminated after warm-reboot.

So I replace fdbclear command with swssconfig DEL.

### Type of change

- [ ] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [x] Test case(new/improvement)

#### How did you do it?
Use j2 template to render FDB JSON config and apply with swssconfig.
#### How did you verify/test it?

Run the original test file "crm_test_fdb_entry.yml" first.
Check AppDB with redis-cli after testing , there is a record "FDB_TABLE:Vlan2:52-54-00-25-07-E9" left.

Apply the patch and run test file "crm_test_fdb_entry.yml" again.
Check AppDB with redis-cli after testing , the is no testing  record left.

### Documentation

### CODE REVIEW

**Comment by lguohan** [comment]
"However with fdbclear the record is not removed from appDB after testing.", why is this? is this a sonic bug, or test issue?

File: ansible/roles/test/tasks/crm/crm_test_fdb_entry.yml [hunk 1]
```diff
@@ -13,14 +13,16 @@
         crm_stats_fdb_entry_used: "{{ out.stdout_lines[0] }}"
         crm_stats_fdb_entry_available: "{{ out.stdout_lines[1] }}"
 
-    - name: Copy FDB JSON config to switch.
-      copy: src=roles/test/tasks/crm/fdb.json dest=/tmp
+    - set_fact: op="SET"
+
+    - name: Render FDB JSON config.
+      template: src=roles/test/templates/fdb_conf.j2 dest=/tmp/fdb_conf.json
```
Comment (lguohan) [inline ansible/roles/test/tasks/crm/crm_test_fdb_entry.yml:19]: you rename the file to fdb_json.j2 but here you use fdb_conf.j2, obviously, there is a mismatch of the name. Can you provide your test run results? We would expect you have run the test successfully before you submit the fix.

**Comment by vincentchiang-ec** [comment]
fdbclear removes FDB entry from ASIC and syncd notify fdborch to remove entry from stateDB. The record in appDB looks like user configuration.

**Comment by skshih** [comment]
I think it's test issue, the fdbclear behavior just following the design document.

https://github.com/Azure/SONiC/blob/d4dacee69ac8eb51e22a11a6ae08d210c42d812e/doc/layer2-forwarding-enhancements/images/portVlanRemoveFlush.jpg

**Comment by skshih** [comment]
I run the crm test case, entire test result is in [crmlog.zip](https://github.com/Azure/sonic-mgmt/files/4502515/crmlog.zip)

I just list the test summary here.

ok=2671 changed=768  unreachable=0    failed=0    skipped=920  rescued=0    ignored=0   

test : Make sure CRM counters updated -------------------------------------------------------------------------- 30.28s
/var/sonic/sonic-mgmt/ansible/roles/test/tasks/crm.yml:24 -------------------------------------------------------------
test : Make sure CRM counters updated --------------------------------------------------------------------------- 4.30s
/var/sonic/sonic-mgmt/ansible/roles/test/tasks/crm/crm_test_fdb_entry.yml:35 ------------------------------------------
test : Make sure CRM counters updated --------------------------------------------------------------------------- 4.30s
/var/sonic/sonic-mgmt/ansible/roles/test/tasks/crm/crm_test_ipv4_neighbor.yml:34 --------------------------------------
test : Make sure CRM counters updated --------------------------------------------------------------------------- 4.30s
/var/sonic/sonic-mgmt/ansible/roles/test/tasks/crm/crm_test_ipv4_nexthop.yml:34 ---------------------------------------
test : Make sure CRM counters updated --------------------------------------------------------------------------- 4.29s
/var/sonic/sonic-mgmt/ansible/roles/test/tasks/crm/crm_test_nexthop_group.yml:62 --------------------------------------
test : Make sure CRM counters updated --------------------------------------------------------------------------- 4.29s
/var/sonic/sonic-mgmt/ansible/roles/test/tasks/crm/crm_test_acl_counter.yml:38 ----------------------------------------
test : Make sure CRM counters updated --------------------------------------------------------------------------- 4.29s
/var/sonic/sonic-mgmt/ansible/roles/test/tasks/crm/crm_test_nexthop_group_member.yml:52 -------------------------------
test : Make sure CRM counters updated --------------------------------------------------------------------------- 4.29s
/var/sonic/sonic-mgmt/ansible/roles/test/tasks/crm/crm_test_ipv4_neighbor.yml:14 --------------------------------------
test : Make sure CRM counters updated --------------------------------------------------------------------------- 4.28s
/var/sonic/sonic-mgmt/ansible/roles/test/tasks/crm/crm_test_ipv4_nexthop.yml:14 ---------------------------------------
test : Make sure CRM counters updated --------------------------------------------------------------------------- 4.28s
/var/sonic/sonic-mgmt/ansible/roles/test/tasks/crm/crm_test_nexthop_group_member.yml:32 -------------------------------
test : Make sure CRM counters updated --------------------------------------------------------------------------- 4.28s
/var/sonic/sonic-mgmt/ansible/roles/test/tasks/crm/crm_test_nexthop_group.yml:32 --------------------------------------
test : Make sure CRM counters updated --------------------------------------------------------------------------- 4.28s
/var/sonic/sonic-mgmt/ansible/roles/test/tasks/crm/crm_test_ipv4_route.yml:23 -----------------------------------------
test : Make sure CRM counters updated --------------------------------------------------------------------------- 4.28s
/var/sonic/sonic-mgmt/ansible/roles/test/tasks/crm/crm_test_acl_entry.yml:36 ------------------------------------------
test : Make sure CRM counters updated --------------------------------------------------------------------------- 4.28s
/var/sonic/sonic-mgmt/ansible/roles/test/tasks/crm/crm_test_ipv6_nexthop.yml:14 ---------------------------------------
test : Make sure CRM counters updated --------------------------------------------------------------------------- 4.28s
/var/sonic/sonic-mgmt/ansible/roles/test/tasks/crm/crm_test_fdb_entry.yml:69 ------------------------------------------
test : Make sure CRM counters updated --------------------------------------------------------------------------- 4.28s
/var/sonic/sonic-mgmt/ansible/roles/test/tasks/crm/crm_test_acl_entry.yml:62 ------------------------------------------
test : Make sure CRM counters updated --------------------------------------------------------------------------- 4.28s
/var/sonic/sonic-mgmt/ansible/roles/test/tasks/crm/crm_test_ipv6_route.yml:28 -----------------------------------------
test : Make sure CRM counters updated --------------------------------------------------------------------------- 4.27s
/var/sonic/sonic-mgmt/ansible/roles/test/tasks/crm/crm_test_acl_counter.yml:64 ----------------------------------------
test : Make sure CRM counters updated --------------------------------------------------------------------------- 4.27s
/var/sonic/sonic-mgmt/ansible/roles/test/tasks/crm/crm_test_acl_entry.yml:13 ------------------------------------------
test : Make sure CRM counters updated --------------------------------------------------------------------------- 4.27s
/var/sonic/sonic-mgmt/ansible/roles/test/tasks/crm/crm_test_ipv6_neighbor.yml:14 --------------------------------------

**Comment by lguohan** [comment]
i understand the issue here. i am discussing with the team what is the proper way to solve this issue.

**Comment by skshih** [comment]
@lguohan May I ask your opinion or plan about this issue after your discussion ?

**Pull request closed without merging**